src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -50,11 +50,14 @@
     protected Text nameText;
     protected HBox nameBox;
     protected HBox menuBarNameArea;
+    protected HBox menuBarCloseButton;
     protected Label renameButton;
     protected PanelNameTextField renameTextField;
     protected FilterTextField filterTextField;
     
     private String panelName = "Panel";
+    private static final int TEXT_PANEL_WIDTH = 385;
```

Comment:
PANEL_WIDTH is a constant in AbstractPanel with value 400. Define this (and the other constant) in terms of that so the width of the field will increase if we ever change the width of the panel.

Also TEXT_PANEL_WIDTH isn't the most specific name for the width of a text field.